formatter/formatter.go

```diff
@@ -0,0 +1,56 @@
+package formatter
+
+import (
+	"github.com/netsampler/goflow2/v2/format"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	PROTO_REMAPPED_IN_BYTES  = 200
+	PROTO_REMAPPED_IN_PKTS   = 201
+	PROTO_REMAPPED_OUT_BYTES = 202
+	PROTO_REMAPPED_OUT_PKTS  = 203
+)
+
+// goflow2 allows some remapping of fields. We use this trick to save IN_BYTES/PKTS
+// and OUT_BYTES/PKTS into custom fields in the protobuf message so that are not lost.
+var MappingYamlStr = `
```

Comment:
personally, i'd use https://pkg.go.dev/embed for this file as it will be easier to maintain over time and people will have syntax highlighting which helps prevent errors

Reply:
fixed.